src/components/TImeLeftBar/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react'
+import { Progress } from 'antd'
+import PropTypes from 'prop-types'
+
+export const TimeLeftBar = ({
+  duration,
+  colorone,
```

Comment:
Props should follow camel casing
- colorOne
- colorTwo
- colorThreshold

Reply:
Updated the variable names